builds/msvc/vs2008/libzmq/libzmq.vcproj

```diff
@@ -121,6 +121,7 @@
       <File RelativePath="..\..\..\..\src\kqueue.cpp" />
       <File RelativePath="..\..\..\..\src\lb.cpp" />
       <File RelativePath="..\..\..\..\src\mailbox.cpp" />
```

Comment:
The VS2008 project doesn't even contain curve_client.cpp and curve_server.cpp ...